python/cudf/benchmarks/API/bench_functions.py

```diff
@@ -50,3 +52,28 @@ def bench_get_dummies_simple(benchmark, prefix):
     benchmark(
         cudf.get_dummies, df, columns=["col1", "col2", "col3"], prefix=prefix
     )
+
+
+@benchmark_with_object(cls="dataframe", dtype="int", cols=6)
+def bench_pivot_table_simple(benchmark, dataframe):
+    values = ["d", "e"]
+    index = ["a", "b"]
+    columns = ["c"]
+    benchmark(
+        cudf.pivot_table,
+        data=dataframe,
+        values=values,
+        index=index,
+        columns=columns,
+    )
+
+
+@pytest_cases.parametrize("nr", NUM_ROWS)
+def bench_crosstab_simple(benchmark, nr):
+    series_a = np.array(["foo", "bar"] * nr)
+    series_b = np.array(["one", "two"] * nr)
+    series_c = np.array(["dull", "shiny"] * nr)
+    np.random.shuffle(series_a)
+    np.random.shuffle(series_b)
+    np.random.shuffle(series_c)
+    benchmark(cudf.crosstab, index=series_a, columns=[series_b, series_c])
```

Comment:
Benchmarks should start from GPU-resident data. We shouldn't benchmark the overhead of host-device transfer and building the `cudf.Series` here. Can you make `cudf.Series` objects before calling `benchmark`?

Reply:
Done.